bundle/edu.gemini.oodb.too.window/src/main/java/edu/gemini/too/email/osgi/OsgiEmailConfig.java

```diff
@@ -74,7 +68,12 @@ public Site getSite() {
     }
 
     public URL getEmailConfig() {
-        return getClass().getClassLoader().getResource("/resources/emailConf.xml");
+        // If we are testing, use the testing email configurations.
+        // If we are in production, use the production email configurations.
+        if (Version.current.isTest())
+            return getClass().getClassLoader().getResource("resources/emailConf-test.xml");
+        else
+            return getClass().getClassLoader().getResource("/resources/emailConf.xml");
```

Comment:
Wouldn't we need `emailConf.xml` for the distributions?

Reply:
Before it was just one file taken from the classpath.  The only change is to add another one, with testing emails that we will use if the version is a test version.  I may be missing something though?

Reply:
I don't think so? Previously there was only one for all distributions, and this just splits it into two depending on whether the test flag is for the current `Version`.